contracts/contracts/ccip/onramp/contract.tolk

```diff
@@ -456,6 +451,23 @@ fun onUpgrade(msg: Upgradeable_Upgrade) {
     Upgradeable{ migrate, version}.upgrade(msg);
 }
 
+fun onWithdrawFeeTokens(msg: OnRamp_WithdrawFeeTokens) {
+    val feeTokens = msg.feeTokens.iter();
+    assert (feeTokens.empty()) throw Error.UnknownToken;
```

Comment:
Let's please document why is this now blocked and how we expect this fn to evolve.